main.py

```
@@ -1,44 +1,43 @@
+    "kangle": "95-96康乐会所部长",
+    "gongming": "光明链接",
+    "jinyu": "金鱼",
+}
+
+async def start(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    code = context.args[0] if context.args else None
+    user_id = update.effective_user.id
+
+    if code and code in name_map:
+        chinese = name_map[code]
+        print(f"用户 {user_id} 通过 {code}（{chinese}）进入")
```

Ruff (0.15.17)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)